### DISCUSSION TITLE
How to calculate the volume of the alpha shape 3D

### DISCUSSION THREAD

**Discussion by Kasi** [CGAL/cgal#7553] [General]
_Please use the following template to help us solving your issue._

## Issue Details

_Describe your issue. Please be specific (compilation error, runtime error, unexpected behavior, wrong results, etc.)._

## Source Code

_If your issue arises by using CGAL in your own source code, please provide a minimalist example that we can compile easily to reproduce the bug. If your issue arises from using a CGAL program (demo, example, etc.), please let us know which one. Helping you solving an issue is much easier and efficient if we can reproduce it._

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits):
* Compiler:
* Release or debug mode:
* Specific flags used (if any):
* CGAL version: 5.5.2
* Boost version:
* Other libraries versions if used (Eigen, TBB, etc.):

**Comment by Kasi** [comment]
I have built an object using the type Alpha _ shape _ 3. How do I calculate the volume of that object?

**Comment by afabri** [comment]
Iterate over the cells  in the alpha shape using this [iterator](https://doc.cgal.org/latest/Alpha_shapes_3/classCGAL_1_1Alpha__shape__3.html#a2fc51e1699b61b18efdeb9d7083c13b5) and sum up the volume of the tetrahedra

**Comment by stla** [comment]
See [my code here](https://github.com/stla/AlphaHull3D/blob/main/src/alphahull3d.cpp) (`vol1` and `vol2`).
